Sources/Routing/Models/Router.swift

```diff
@@ -9,6 +9,8 @@ import SwiftUI
 public typealias Routable = NavigationPathManageable & SheetManageable & FullScreenCoverManageable & AlertManageable & ToastManageable
 
 public final class Router<Routes: ViewDisplayable>: Routable {
+    
+    public var onDismiss: CallBackHandler?
```

Comment:
We will need one for fullScreenCover too..
So maybe have one named `onSheetDismiss` and the other `onFullScreenCoverDismiss` ?